andela/Ah-backend-aquaman
# #164069233 Pagination functionality for articles

#### What does this PR do?
- Add pagination functionality for articles 
#### Description of Task to be completed?
- Create a pagination module to override default settings.
- Add a pagination class in views
#### How should this be manually tested?
- Clone the [repo](https://github.com/andela/Ah-backend-aquaman), cd into `Ah-backend-aquaman` and checkout the branch `ft-pagination-164069233`
    -  pip install -r requirements.txt
    -  python manage.py migrate
    -  python manage.py runserver
    -  create an account, verify your email, login and create articles
    -  use postman to test the pagination endpoint `/api/articles/`
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#164069233](https://www.pivotaltracker.com/story/show/164069233)

#### Screenshots (if appropriate)
<img width="1203" alt="Screenshot 2019-03-14 at 14 29 59" src="https://user-images.githubusercontent.com/6951438/54353679-b2fbd300-4665-11e9-98dc-2c23a853de91.png">
